### PR TITLE
Refactor some bits of code

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: src/Integration.Vsix.UnitTests/SonarLintDaemonTests.cs [hunk 1]
```diff
@@ -40,56 +61,55 @@ public void Run_Without_Install()
...
             }
 
-            foreach (string dir in directories)
+            foreach (string dir in Directory.EnumerateDirectories(path))
             {
                 ForceDeleteDirectory(dir);
             }
 
-            Directory.Delete(path, true);
+            Thread.Sleep(1);
```
Comment (janos-ss) [inline src/Integration.Vsix.UnitTests/SonarLintDaemonTests.cs]: This fixes the file locking problem that made you comment out some lines last week?

File: src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs [hunk 2]
```diff
@@ -18,10 +18,10 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using Microsoft.VisualStudio.Shell;
+using System;
 using System.Windows;
 using System.Windows.Controls;
-using System;
+using Microsoft.VisualStudio.Shell;
```
Comment (janos-ss) [inline src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs]: The **Remove and Sort Usings** menu option does it for me the way it was.

File: src/Integration.Vsix.UnitTests/SonarLintDaemonTests.cs (hunk 1 above)
Comment (Evangelink) [inline src/Integration.Vsix.UnitTests/SonarLintDaemonTests.cs]: I read that it was supposed to make it but......CI build says nope/

File: src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs (hunk 2 above)
Comment (Evangelink) [inline src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs]: It is an arbitrary decision of @valhristov and I but we have checked the option to put the `System` using first. Options -> Text Editor -> C# -> Advanced, check the `Place 'System' directives first when sorting usings`.

Comment (janos-ss) [inline src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs]: Good to know, updated my settings.

File: src/Integration.Vsix.UnitTests/SonarLintDaemonTests.cs (hunk 1 above)
Comment (janos-ss) [inline src/Integration.Vsix.UnitTests/SonarLintDaemonTests.cs]: Bummer eh :shit:

**Pull request merged**